dojo/tasks.py

```diff
@@ -184,26 +184,27 @@ def async_custom_pdf_report(self,
             x = urlencode({'title': cp.title,
                            'subtitle': cp.sub_heading,
                            'info': cp.meta_info})
-            cover = host + reverse(
+            cover = "http://nginx:8080/" + reverse(
```

Comment:
Are we sure this hardcoded url is what we want here?

Reply:
Agree, it should be pulled from settings.py and with a default value.

Reply:
Sure, I can do that. What nginx:8080 does in this case is allow for the celery worker to reach the nginx container by resolving its IP automatically. What would you like the parameter to be listed as under settings?

Reply:
Moved into settings. Also added param in settings to switch between containerized and non-containerized path. At the moment this only matters for things that run in the celery_worker that need to hit nginx.